### PR TITLE
Updated submitReviews to use axios 

### PR DESCRIPTION
**Pull request by juliejeong**
### Summary 

This pull request updates the function **submitReviews** from Forms.jsx (client) to use axios and renames the diffculty field of the object **newReviews** from _diff_ to _difficulty_ to match the fields.

### Test Plan 

Test locally - I submitted a review and checked that it went through successfully

### CODE REVIEW

File: client/src/ui/Form.jsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import './css/Form.css';
 import { Session } from '../meteor-session';
 import { Meteor } from "../meteor-shim";
```
Comment (khemritolya) [inline client/src/ui/Form.jsx:10]: Just remove this line here.

**Review by khemritolya — COMMENTED** [review]
Looks good! There's one last change I'd recommend you make, which is to remove the meteor shim import. We don't need it anymore.

File: client/src/ui/Form.jsx [hunk 2]
```diff
@@ -9,6 +9,7 @@ import './css/Form.css';
 import { Session } from '../meteor-session';
```
Comment (khemritolya) [inline client/src/ui/Form.jsx:9]: Not this line.

File: client/src/ui/Form.jsx (hunk 1 above)
Comment (juliejeong) [reply client/src/ui/Form.jsx]: I just did and pushed the branch with the new commit!

**Review by khemritolya — COMMENTED** [review]
One last quick thing -- please merge in the changes by akash to master!

**Review by khemritolya — APPROVED** [review]
Looks good!

**Pull request merged**